compatibility with colorscheme

hdoi opened the pull request:
The original syntax had a fixed color for specification. The color specification was made to follow the colorscheme.

Commit:
compatibility with colorscheme

The original syntax had a fixed color for specification.
The color specification was made to follow the colorscheme

ccwang002 reviewed (changes requested):
Hi @hdoi, thank you for taking the time to submit this PR and for your efforts to improve this plugin. I haven’t worked with PDB files recently, so I appreciate your help in ensuring everything continues to work smoothly with these updates :)

From what I gather, the main enhancement here is letting the plugin use Vim’s color scheme settings for color values. The theme example screenshots you provided are fantastic, thank you! Could you please also update the README with a brief explanation and example instructions? For example, I was wondering if you could mention that users can change the highlighting by switching Vim color schemes (e.g., with `:colorscheme desert`). If you could update the README, I will be very happy to merge this PR.

I also added Copilot to review this PR but I don't know if the syntax color mapping is making sense. I will trust your judgement. 

Thanks again!

ccwang002 commented on README.md:
```diff
@@ -1,3 +1,10 @@
+![3](./image/3.png "3")
+![4](./image/4.png "4")
+![5](./image/5.png "5")
+![6](./image/6.png "6")
```
This is optional

ccwang002 commented on syntax/pdb.vim:
```diff
@@ -63,13 +65,13 @@ syn match Connect_4  /\%>26v.*\%32v/ contained nextgroup=Connect_e
+hi def link Connect_2  Function
+hi def link Connect_3  Float
+hi def link Connect_4  Function
+hi def link Connect_e  Float
```
This is more like Connect (extra fields). Do you think we should map it to a different color?

hdoi replied:
Maybe we should, but Error is too emphatic. Many color schemes really emphasize Error, so I would go with a more moderate color.

Commit:
Update README and refine CONECT_e color

Update README to include new color customization.
Improved the visibility of CONECT_e color.

ccwang002 reviewed (approved):
LGTM Thank you again for your contribution!!

hdoi commented:
Thanks a lot for merging my PR!
